### PR TITLE
Menus: Feature example in the jetpack manage error screen

### PR DESCRIPTION
**Pull request by johnHackworth**
This PR adds a feature example showing the menus screento the error screen showed to the jetpack sites which doesn't have a recent version of jetpack:

Before:
image

After:
image

ping @enejb & @rickybanister

### CODE REVIEW

**Comment by rickybanister** [comment]
Looks right to me..

cc @folletto and @alternatekev

**Comment by alternatekev** [comment]
Looks good except the same issue as themes where the buttons are in the wrong order.

**Comment by folletto** [comment]
:+1:

**Comment by johnHackworth** [comment]
as mentioned here https://github.com/Automattic/wp-calypso/pull/2052#issuecomment-168997565, the order of the buttons is the one that is used in `emptyContent`... if we change it, it would be better done in an independent PR just for it

File: client/my-sites/menus/main.jsx [hunk 1]
```diff
@@ -206,6 +206,16 @@ var Menus = React.createClass( {
 	},
 
 	renderJetpackManageDisabledMessage: function( site ) {
+		const data = this.props.siteMenus.get();
+		let featureExample;
+
```
Comment (enejb) [inline client/my-sites/menus/main.jsx]: An extra new line.

File: client/my-sites/menus/main.jsx [hunk 2]
```diff
@@ -206,6 +206,16 @@ var Menus = React.createClass( {
...
+
+		if ( data.menus && data.locations &&
+			data.hasDefaultMenu && this.props.itemTypes.fetched ) {
+			featureExample = this.renderMenus();
+		}
+
```
Comment (enejb) [inline client/my-sites/menus/main.jsx]: here as well. ( extra new line)

**Comment by enejb** [comment]
I found a bug: 
screen shot 2016-01-05 at 11 30 44

When you are on the page with manage disabled and you click on the new post icon. Top right pen icon. The feature example overlay has a higher z index and prevents me from selecting my sites. 

I agree with @johnHackworth that the changing the order of the buttons can be done in a different PR.

**Comment by johnHackworth** [comment]
fixed, @enejb :
image

**Comment by enejb** [comment]
Works as expected. Thanks for the fixes.

**Pull request merged**